### PR TITLE
i18n(es): Update decap-cms.mdx

### PR DESCRIPTION
**Pull request by Eric-llos**
Updated `decap-cms.mdx` for the ES translation.

I didn't know how to manage the <tags></tags> and if needed to translate them so i made a mixt and translated only the comments :/ Any feedback is appreciated!

Thanks

### CODE REVIEW

File: src/content/docs/es/guides/cms/decap-cms.mdx [hunk 1]
```diff
@@ -10,6 +10,121 @@ import Grid from '~/components/FluidGrid.astro'
 import Card from '~/components/ShowcaseCard.astro'
 
 [Decap CMS](https://www.decapcms.org/) (anteriormente Netlify CMS) es un sistema de gestión de contenido de código abierto basado en Git.
+
+Decap te permite aprovechar al máximo todas las funciones de Astro, incluida la optimización de imágenes y las colecciones de contenido.
```
Comment (sarah11918) [inline src/content/docs/es/guides/cms/decap-cms.mdx:14]:
```suggestion
import Grid from '~/components/FluidGrid.astro'
import Card from '~/components/ShowcaseCard.astro'
import { FileTree } from '@astrojs/starlight/components'
import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'

[Decap CMS](https://www.decapcms.org/) (anteriormente Netlify CMS) es un sistema de gestión de contenido de código abierto basado en Git.

Decap te permite aprovechar al máximo todas las funciones de Astro, incluida la optimización de imágenes y las colecciones de contenido.
```
This should help the checks! More components to import here now.

File: src/content/docs/es/guides/cms/decap-cms.mdx [hunk 2]
```diff
@@ -7,9 +7,127 @@ service: Decap CMS
 i18nReady: true
 ---
 import Grid from '~/components/FluidGrid.astro'
+import Grid from '~/components/FluidGrid.astro'
```
Comment (sarah11918) [inline src/content/docs/es/guides/cms/decap-cms.mdx:10]:
```suggestion
```

**Comment by sarah11918** [comment]
Hi @Eric-llos ! I'm adding the imports at the top of the file to see whether that fixes the problem. So, if you are also working on this locally again (instead of just committing suggestions here on GitHub from reviews) make sure that your local copy is updated!

File: src/content/docs/es/guides/cms/decap-cms.mdx [hunk 3]
```diff
@@ -8,8 +8,125 @@ i18nReady: true
...
+
+Decap CMS fue desarrollado originalmente por Netlify y tiene soporte de primera clase para [Netlify Identity](https://docs.netlify.com/security/secure-access-to-sites/identity/).
+
+Al implementar en netlify, configura la identidad para tu proyecto a través del panel de Netlify e incluye el [Netlify Identity Widget](https://github.com/netlify/netlify-identity-widget) en la ruta `admin` de tu proyecto. Opcionalmente, incluye el widget de identidad en la página de inicio de tu sitio si planeas invitar a nuevos usuarios por correo electrónico.
+
+### Decap CMS con clientes OAuth externos
+
+Al implementar en proveedores de hosting distintos de Netlify, debes crear tus propias rutas OAuth.
+
+En Astro, esto se puede hacer con rutas renderizadas bajo demanda en tu proyecto configurado con [salida `servidor` o `híbrida`](/es/guides/server-side-rendering/#enable-on-demand-server-rendering ) activado.
```
Comment (sarah11918) [inline src/content/docs/es/guides/cms/decap-cms.mdx:126]:
```suggestion
En Astro, esto se puede hacer con rutas renderizadas bajo demanda en tu proyecto configurado con [salida `servidor` o `híbrida`](/es/guides/server-side-rendering/habilita-la-renderización-de-servidor-bajo-demanda) activado.
```
Hopefully this is the last fix needed!

We got past the imports, but any links to other pages in docs need to also have translated section headers if they exist!  :raised_hands:

File: src/content/docs/es/guides/cms/decap-cms.mdx [hunk 4]
```diff
@@ -8,17 +8,134 @@ i18nReady: true
...
+
+Decap CMS fue desarrollado originalmente por Netlify y tiene soporte de primera clase para [Netlify Identity](https://docs.netlify.com/security/secure-access-to-sites/identity/).
+
+Al implementar en netlify, configura la identidad para tu proyecto a través del panel de Netlify e incluye el [Netlify Identity Widget](https://github.com/netlify/netlify-identity-widget) en la ruta `admin` de tu proyecto. Opcionalmente, incluye el widget de identidad en la página de inicio de tu sitio si planeas invitar a nuevos usuarios por correo electrónico.
+
+### Decap CMS con clientes OAuth externos
+
+Al implementar en proveedores de hosting distintos de Netlify, debes crear tus propias rutas OAuth.
+
+En Astro, esto se puede hacer con rutas renderizadas bajo demanda en tu proyecto configurado con [salida `servidor` o `híbrida`](/es/guides/server-side-rendering/habilita-la-renderización-de-servidor-bajo-demanda) activado.
```
Comment (mingjunlu) [inline src/content/docs/es/guides/cms/decap-cms.mdx:126]: Caught a typo here. The hashtag `#` was missing

```suggestion
En Astro, esto se puede hacer con rutas renderizadas bajo demanda en tu proyecto configurado con [salida `servidor` o `híbrida`](/es/guides/server-side-rendering/#habilita-la-renderización-de-servidor-bajo-demanda) activado.
```

**Comment by Eric-llos** [comment]
Thanks @sarah11918 and @mingjunlu for your feedback and help :)

File: src/content/docs/es/guides/cms/decap-cms.mdx [hunk 5]
```diff
@@ -8,17 +8,134 @@ i18nReady: true
...
+      ```
+      </Fragment>
+      <Fragment slot="yarn">
+      ```shell
+      yarn add decap-cms-app
+      ```
+      </Fragment>
+    </PackageManagerTabs>
+
+2. Importa el paquete a una etiqueta `<script>` en su página `<body>`
```
Comment (dreyfus92) [inline src/content/docs/es/guides/cms/decap-cms.mdx:44]:
```suggestion
2. Importa el paquete a una etiqueta `<script>` dentro del `<body>` de tu página.
```

**Review by dreyfus92 — COMMENTED** [review]
Awesome job @Eric-llos, I just left a small suggestion 😁

File: src/content/docs/es/guides/cms/decap-cms.mdx [hunk 6]
```diff
@@ -8,17 +8,134 @@ i18nReady: true
...
+         fields: # Los campos para cada documento, generalmente al principio
+            - { label: "Layout", name: "layout", widget: "hidden", default: "blog" }
+            - { label: "Title", name: "title", widget: "string" }
+            - { label: "Publish Date", name: "date", widget: "datetime" }
+            - { label: "Featured Image", name: "thumbnail", widget: "image" }
+            - { label: "Rating (scale of 1-5)", name: "rating", widget: "number" }
+            - { label: "Body", name: "body", widget: "markdown" }
+     ```
+
+4. Agrega la ruta `admin` para tu aplicación React. Este archivo puede ser `public/admin/index.html` junto con `config.yml` o, si prefieres utilizar una ruta Astro, `src/pages/admin.astro`.
```
Comment (Waxer59) [inline src/content/docs/es/guides/cms/decap-cms.mdx:79]:
```suggestion
4. Agregar la ruta `admin` para tu aplicación React. Este archivo puede ser `public/admin/index.html` junto con `config.yml` o, si prefieres utilizar una ruta de Astro, `src/pages/admin.astro`.
```

File: src/content/docs/es/guides/cms/decap-cms.mdx [hunk 7]
```diff
@@ -8,17 +8,134 @@ i18nReady: true
...
+        <link href="/admin/config.yml" type="text/yaml" rel="cms-config-url" />
+        <title>Administrador de contenidos</title>
+       </head>
+      <body>
+        <script src="https://unpkg.com/decap-cms@^3.1.2/dist/decap-cms.js"></script>
+      </body>
+    </html>
+    ```
+
+5. Para habilitar la carga de medios a una carpeta específica a través del editor Decap, agregue una ruta adecuada:
```
Comment (Waxer59) [inline src/content/docs/es/guides/cms/decap-cms.mdx:103]:
```suggestion
5. Para habilitar la carga de medios a una carpeta específica a través del editor de Decap, agrega una ruta adecuada:
```

File: src/content/docs/es/guides/cms/decap-cms.mdx [hunk 8]
```diff
@@ -8,17 +8,134 @@ i18nReady: true
...
+     ```yml title="/public/admin/config.yml"
+     media_folder: "src/assets/images" # Ubicación donde se almacenarán los archivos en el repositorio
+     public_folder: "src/assets/images" # El atributo src para los medios cargados
+     ```
+
+Consulta la [documentación de configuración de Decap CMS](https://decapcms.org/docs/configure-decap-cms/) para obtener instrucciones y opciones completas.
+
+## Uso
+Navega hasta `yoursite.com/admin/` para usar el editor Decap CMS.
```
Comment (Waxer59) [inline src/content/docs/es/guides/cms/decap-cms.mdx:112]:
```suggestion
Navega hasta `yoursite.com/admin/` para usar el editor de Decap CMS.
```

File: src/content/docs/es/guides/cms/decap-cms.mdx [hunk 9]
```diff
@@ -8,17 +8,134 @@ i18nReady: true
...
+
+## Autenticación
+
+### Decap CMS con Netlify Identity
+
+Decap CMS fue desarrollado originalmente por Netlify y tiene soporte de primera clase para [Netlify Identity](https://docs.netlify.com/security/secure-access-to-sites/identity/).
+
+Al implementar en netlify, configura la identidad para tu proyecto a través del panel de Netlify e incluye el [Netlify Identity Widget](https://github.com/netlify/netlify-identity-widget) en la ruta `admin` de tu proyecto. Opcionalmente, incluye el widget de identidad en la página de inicio de tu sitio si planeas invitar a nuevos usuarios por correo electrónico.
```
Comment (Waxer59) [inline src/content/docs/es/guides/cms/decap-cms.mdx:120]:
```suggestion
Cuando despliegues en netlify, configura la identidad para tu proyecto a través del panel de Netlify e incluye el [Netlify Identity Widget](https://github.com/netlify/netlify-identity-widget) en la ruta `admin` de tu proyecto. Opcionalmente, incluye el widget de identidad en la página de inicio de tu sitio si planeas invitar a nuevos usuarios por correo electrónico.
```

File: src/content/docs/es/guides/cms/decap-cms.mdx [hunk 10]
```diff
@@ -8,17 +8,134 @@ i18nReady: true
...
 import Grid from '~/components/FluidGrid.astro'
 import Card from '~/components/ShowcaseCard.astro'
+import { FileTree } from '@astrojs/starlight/components'
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 
 [Decap CMS](https://www.decapcms.org/) (anteriormente Netlify CMS) es un sistema de gestión de contenido de código abierto basado en Git.
+
+Decap te permite aprovechar al máximo todas las funciones de Astro, incluida la optimización de imágenes y las colecciones de contenido.
+
+Decap añade una ruta (normalmente `/admin`) a tu proyecto que cargará una aplicación React para permitir a los usuarios autorizados administrar el contenido directamente desde el sitio web implementado. Decap enviará los cambios directamente al repositorio fuente de su proyecto Astro.
```
Comment (Waxer59) [inline src/content/docs/es/guides/cms/decap-cms.mdx:18]:
```suggestion
Decap añade una ruta (normalmente `/admin`) a tu proyecto que cargará una aplicación React para permitir a los usuarios autorizados administrar el contenido directamente desde el sitio web desplegado. Decap enviará los cambios directamente al repositorio fuente de su proyecto Astro.
```

File: src/content/docs/es/guides/cms/decap-cms.mdx [hunk 11]
```diff
@@ -8,17 +8,134 @@ i18nReady: true
...
+
+### Decap CMS con Netlify Identity
+
+Decap CMS fue desarrollado originalmente por Netlify y tiene soporte de primera clase para [Netlify Identity](https://docs.netlify.com/security/secure-access-to-sites/identity/).
+
+Al implementar en netlify, configura la identidad para tu proyecto a través del panel de Netlify e incluye el [Netlify Identity Widget](https://github.com/netlify/netlify-identity-widget) en la ruta `admin` de tu proyecto. Opcionalmente, incluye el widget de identidad en la página de inicio de tu sitio si planeas invitar a nuevos usuarios por correo electrónico.
+
+### Decap CMS con clientes OAuth externos
+
+Al implementar en proveedores de hosting distintos de Netlify, debes crear tus propias rutas OAuth.
```
Comment (Waxer59) [inline src/content/docs/es/guides/cms/decap-cms.mdx:124]:
```suggestion
Cuando despliegues en proveedores de hosting distintos de Netlify, debes crear tus propias rutas OAuth.
```

File: src/content/docs/es/guides/cms/decap-cms.mdx [hunk 12]
```diff
@@ -8,17 +8,134 @@ i18nReady: true
...
+
+En Astro, esto se puede hacer con rutas renderizadas bajo demanda en tu proyecto configurado con [salida `servidor` o `híbrida`](/es/guides/server-side-rendering/#habilita-la-renderización-de-servidor-bajo-demanda) activado.
```
Comment (Waxer59) [inline src/content/docs/es/guides/cms/decap-cms.mdx:126]:
```suggestion
En Astro, esto se puede hacer con rutas renderizadas bajo demanda en tu proyecto configurado con la [salida `server` o `hybrid`](/es/guides/server-side-rendering/#habilita-la-renderización-de-servidor-bajo-demanda) activada.
```

File: src/content/docs/es/guides/cms/decap-cms.mdx [hunk 13]
```diff
@@ -8,17 +8,134 @@ i18nReady: true
...
+
+Consulta [Documentos OAuth de Decap](https://decapcms.org/docs/external-oauth-clients/) para obtener una lista de clientes OAuth compatibles mantenidos por la comunidad.
```
Comment (Waxer59) [inline src/content/docs/es/guides/cms/decap-cms.mdx:128]:
```suggestion
Consulta los [Documentos OAuth de Decap](https://decapcms.org/docs/external-oauth-clients/) para obtener una lista de clientes OAuth compatibles mantenidos por la comunidad.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias!

**Review by dreyfus92 — APPROVED** [review]
Thanks everybody for jumping in. LGTM 😁

**Pull request merged**